IdentityCore/src/broker_operation/request/silent_token_request/MSIDBrokerOperationSilentTokenRequest.m

```diff
@@ -97,5 +103,28 @@ - (NSDictionary *)jsonDictionary
     return json;
 }
 
+- (NSString *)getFullRequestThumbprint
+{
+    NSSet *fullThumbprintExcludeSet = [NSSet setWithArray:@[MSID_ACCOUNT_DISPLAYABLE_ID_JSON_KEY,
+                                                            MSID_BROKER_CLIENT_CAPABILITIES_KEY,
+                                                            MSID_BROKER_CLIENT_VERSION_KEY,
+                                                            MSID_BROKER_CLIENT_APP_VERSION_KEY,
+                                                            MSID_BROKER_CLIENT_APP_NAME_KEY,
+                                                            MSID_BROKER_CORRELATION_ID_KEY,
+                                                            MSID_CLIENT_ID_JSON_KEY]];
+    return [MSIDThumbprintCalculator calculateThumbprint:[self jsonDictionary]
+                                            filteringSet:fullThumbprintExcludeSet
+                                         includePolarity:NO];
```

Comment:
this one might be hard to understand by looking at the definition. I recommend making it more readable. For example, you can have two methods - calculateThumbrpint:excludeKeys:
calculateThumbrprint:includeKeys:

Reply:
we would be creating duplicate codes. wouldn't it make sense to rename the params so it's easier to understand than create two methods that share duplicate codes.